tests/src/common/WskCli.java

```diff
@@ -838,14 +852,16 @@ private boolean logsForActionContain(String action, String sRegex, int initialWa
      *            name of the action.
      * @param w
      *            the string to search for
+     * @param since
+     * 			  since parameter for cli. Only activations after this timestamp will be returned.
```

Comment:
There are tabs in there. (several other places aswell). Can you doublecheck why scanCode doesn't work?